lib/worker/base.js

```diff
@@ -175,22 +176,14 @@ const run = async options => {
 	};
 
 	try {
-		for await (const ref of (options.require || [])) {
-			if (typeof ref === 'string') {
-				await load(ref);
-			} else if (Array.isArray(ref)) {
-				const [path, options] = ref;
-
-				const loadedModule = await load(path);
-
-				if (typeof loadedModule === 'function') {
-					await loadedModule(...options);
-				} else if (typeof loadedModule.default === 'function') {
-					const {default: fn} = loadedModule;
-					await fn(...options);
-				} else {
-					channel.send({type: 'non-invokable-require-option'});
```

Comment:
I'm not sure how important it could be to inform users that the `thing` they want to import wont be called at all. For example if they try to use some local file and forgot to export sth that should be called?

It could be better to see an error instead of silent run when something is not working 🤔

Reply:
I don't think it's necessarily the case that you want to execute a function, so making that required behavior isn't great either.